src/components/Footer.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { useState } from 'react';
+import cn from 'classnames';
+import { Todo } from '../types/Todo';
+
+type Props = {
+  todos: Todo[];
+};
+export const Footer : React.FC<Props> = ({ todos }) => {
+  const [filter, setFilter] = useState('All');
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="TodosCounter">
+        {`${todos.filter(el => el.completed === false).length} items left`}
```

Comment:
move this filter to a variable (before `return`). And if you're familiar with `useMemo` it'd nice to wrap filter with this hook